platform/darwin/src/MGLGeometry.h

```diff
@@ -181,4 +186,7 @@ NS_INLINE CLLocationDegrees MGLDegreesFromRadians(CGFloat radians) {
     return radians * 180 / M_PI;
 }
 
+/** Returns mercator projection of a WGS84 coordinate at the specified zoom level. */
+extern MGL_EXPORT MGLMapPoint MGLMapPointForCoordinate(CLLocationCoordinate2D coordinate, double zoomLevel);
```

Comment:
For consistency with other struct constructors and `MKMapPointMake()`, this function should be named `MGLMapPointMake()`.

Reply:
Oops, I goofed up: `MKMapPointMake()` is really simple, just taking an `x` and `y` and sticking it in an array. You were right to give this function a fuller name of `MGLMapPointForCoordinate()`. A simpler `MGLMapPointMake()` would be a good idea, though.